docs/examples/puppeteer_crawler_crawler_utils_snapshot.ts

```diff
@@ -1,12 +1,12 @@
-import { PuppeteerCrawler, puppeteerUtils } from '@crawlee/puppeteer';
+import { PuppeteerCrawler } from '@crawlee/puppeteer';
 
 // Create a PuppeteerCrawler
 const crawler = new PuppeteerCrawler({
-    async requestHandler({ request, page }) {
+    async requestHandler({ request, saveSnapshot }) {
         // Convert the URL into a valid key
         const key = request.url.replace(/[:/]/g, '_');
```

Comment:
i guess we could make this work automatically and make the `key` param optional

Reply:
Maybe instead this example should use the page title and not the url? Or just omit key entirely as it is not required

Reply:
Oh so its not required already, I guess you can ignore me then :] It was more of an idea for implementation than comment for the example.